build/repo.targets

```diff
@@ -126,15 +126,15 @@
   </Target>
 
   <ItemGroup>
-    <JavaBuildFiles Include="signalr-client-$(JavaClientVersion).jar;signalr-client-$(JavaClientVersion)-javadoc.jar;signalr-client-$(JavaClientVersion)-sources.jar;signalr-client-$(JavaClientVersion).pom"/>
+    <JavaBuildFiles Include="signalr-$(JavaClientVersion).jar;signalr-$(JavaClientVersion)-javadoc.jar;signalr-$(JavaClientVersion)-sources.jar;signalr-$(JavaClientVersion).pom"/>
```

Comment:
Could you define the `Jars` and `PomFile` above here, then use those variables for `JavaBuildFiles` so you don't have to duplicate the names